### PR TITLE
fix: let's fix the build

### PR DESCRIPTION
**Pull request by Dominik1999**

### CODE REVIEW

File: playground/package.json [hunk 1]
```diff
@@ -6,14 +6,15 @@
   "dependencies": {
     "@codemirror/theme-one-dark": "^6.1.0",
     "@headlessui/react": "^1.7.7",
-    "@tailwindcss/typography": "^0.5.9",
     "@tailwindcss/forms": "^0.5.3",
+    "@tailwindcss/typography": "^0.5.9",
```
Comment (hackaugusto) [inline playground/package.json:10]: nit: does this line need to move?

File: playground/src/App.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import NavBar from "./components/NavBar";
 import CodingEnvironment from "./pages/CodingEnvironment";
 import InstructionTable from "./pages/InstructionTable";
 
+
```
Comment (hackaugusto) [inline playground/src/App.tsx:6]: nit: remove this space

File: playground/webpack.config.js [hunk 3]
```diff
@@ -39,4 +38,4 @@ module.exports = {
     filename: 'bundle.js',
     path: path.resolve(__dirname, 'dist'),
   },
-};
\ No newline at end of file
+};
```
Comment (hackaugusto) [inline playground/webpack.config.js:41]: we should probably add `pre-commit` here too, lets see if that gets merged on `miden-vm` and then add here too.

**Review by hackaugusto — COMMENTED** [review]
I ran it locally with:

```
git clone 'https://github.com/0xPolygonMiden/examples.git'
cd examples/playground
git checkout dominik_adding_a_sass_loader_for_katex
npm run build:miden
npm install
npm run start
```

and later

```
npm run build
npx webpack
```

the webpack build finished successfully and the dev version worked as intended.

just a few nits

File: playground/package.json (hunk 1 above)
Comment (Dominik1999) [reply playground/package.json:10]: What do you mean?

File: playground/webpack.config.js (hunk 3 above)
Comment (Dominik1999) [reply playground/webpack.config.js:41]: Got it, can we do that in a separate PR? This should just fix the build.

File: playground/package.json (hunk 1 above)
Comment (hackaugusto) [reply playground/package.json:10]: this dep was already set, but a line above. So the change was just moving the dependency one line down. This is not a big deal, but it makes it harder to navigate in the code's history.

**Review by hackaugusto — APPROVED** [review]
LGTM

File: playground/package.json (hunk 1 above)
Comment (Dominik1999) [reply playground/package.json:10]: Got it, good point. I was copying a lot of dependencies for this PR so I forgot about that.

**Pull request merged**